metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -108,12 +108,19 @@ final class Doctor(
       isSemanticdbEnabled: Boolean,
       scala: ScalaTarget
   ): String = {
+
+    def isMaven: Boolean = workspace.resolve("pom.xml").isFile
+    def hint() =
+      if (isMaven)
+        "check how to enable semanticDB on the " +
```

Comment:
```suggestion
        "Enable SemanticDB following instructions on the " +
```

Reply:
fixed, but `enable` not `Enable` since I capitalize below if this is the only sentence.